xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java

```diff
@@ -99,30 +100,11 @@ public OpenOrders getOpenOrders(OpenOrdersParams params) throws IOException {
 
   @Override
   public String placeMarketOrder(MarketOrder mo) throws IOException {
-
-    return placeOrder(OrderType.MARKET, mo, null, null, null);
+    return Long.toString(placeOrder(OrderType.MARKET, mo, null, null, null).orderId);
   }
 
-  @Override
-  public String placeLimitOrder(LimitOrder lo) throws IOException {
```

Comment:
you want to remove support for LimitOrders in binance ?!

Reply:
Look 167 line. It supports, but more useful as a specific case of calling of placeLimitOrder2, which return full new order object as a value

Reply:
line 167 don't change the fact that you are removing an implementation of a method defined in an interface which is the main API of this library and is used by most users